### ISSUE TITLE
Extensions don't load after upgrading to v1.27.2

### ISSUE THREAD

**Issue by mrcasals** [Microsoft/vscode#58785]
- VSCode Version: 1.27.2, also tried on Insiders 1.28.0
- OS Version: Mac 10.13.6

Steps to Reproduce:

After upgrading to 1.27.2 extensions don't seem to load. For example, GitLens or VS Live Share don't show up in the sidebar, the Vim extension is not working anymore, etc. Here's an screenshot of those extensions being installed and enabled, but not appearing in the sidebar:

![](https://i.imgur.com/Se8k47S.png)

Does this issue occur when all extensions are disabled?: No (misses the point)

I don't know how to help debugging this. I tried searching the issues list and couldn't find anything related.

Thanks!

**Comment by mrcasals** [comment]
I tried downloading versions 1.27.0 and 1.27.1, but extensions still don't load.

This is the Developer Tools screen I'm getting:

![](https://i.imgur.com/IcivEVE.png)

**Comment by sandy081** [comment]
@mrcasals Can you please right click on the activity bar (side bar) and let us know what you see?

**Comment by mrcasals** [comment]
@sandy081 this is what I see:

![](https://i.imgur.com/RlDfP4k.png)

**Comment by sandy081** [comment]
@mrcasals Can you please check if you configured Gitlens to be shown in SCM viewlet rather as separate viewlet?

Not sure if there is a similar setting for Live share?

Can you also try other extensions like `Docker` that contributes view to sider bar?

**Comment by mrcasals** [comment]
@sandy081 I didn't modify any settings related to this. On the first screenshot I posted you can see how the status bar is missing the "Share" button from VS Live Share. Other extensions have stopped working too, like the Vim one.

Also, it looks like VSCode is failing to detect Git for any given open folder, could this be related? Here are the steps I took:

1. Click on the File menu, and open a folder that is also a git project
2. Click on the Source Control button on the side bar
3. See this:

![](https://i.imgur.com/ASUOrcj.png)

For the `docker` extension: I installed it and now I can see it on the sidebar:

![](https://i.imgur.com/jKyU70m.png)

If I try to uninstall and reinstall VS Live Share or GitLens, they don't appear...

**Comment by sandy081** [comment]
Can you please run the command `Show Running Extensions` and let us know what you see?

**Comment by mrcasals** [comment]
@sandy081 I ran it and the screen was completely blank. I realized I have been using the laptop without restarting it for quite a few days now, I restarted the laptop... And extensions now magically work!

![](https://i.imgur.com/4SaFm8a.png)

The "Source Control" tab works as expected, the status bar shows the extensions there... I think it's weird I had to restart the laptop to make this work, but hey, it works.

This issue can be closed, unless you want to keep it open to try to figure out what was happening 😄 

Thanks for your time!
